### DISCUSSION TITLE
申请退款

### DISCUSSION THREAD

**Discussion by Raymondlee321** [getlantern/lantern-forum-cn#829] [General]
今天购买了一年的专业版，现在用别的VPN上来投诉，申请退款，也是服了！
满心欢喜地觉得可以连接三个设备，这下一个都连不上。手机关联不了，更别指望家里的那台电脑了。
申请退款，账号：limian321@gmail.com

**Comment by Raymondlee321** [comment]
你这论坛也是搞笑啊，需要用VPN登录才能上来的论坛，才能反馈问题。
既然是反馈问题的，为什么不是针对当地网络能够登录的呢？

**Comment by MaxMKMax** [comment]
@Raymondlee321 您好. 那我们帮您提交退款申请. 请您也提供一下支付单据, 然后我才能帮您提交问题. 谢谢. 

请问您所说的连不了是网络问题还是收不到授权码? 

请您提供我们log. 我们看是出现什么问题. 暂时没有别的用户遇到像您的问题. 我们需要log提交给技术部门. 帮我们提交各应用内报告吧. 然后给我 6 位数票号. 谢谢!
